MemorialHouse/MHPresentation/MHPresentation/Source/EditBook/View/EditBookViewController.swift

```diff
@@ -212,23 +201,27 @@ final class EditBookViewController: UIViewController {
             name: UIResponder.keyboardWillHideNotification,
             object: nil
             )
+        editPageTableView.keyboardDismissMode = .onDrag
     }
     private func configureBinding() {
-        // TODO: - 추후 로직 추가하기
         let output = viewModel.transform(input: input.eraseToAnyPublisher())
         output.receive(on: DispatchQueue.main)
             .sink { [weak self] event in
                 switch event {
-                case .setTableView:
+                case .updateViewController(title: let title):
```

Comment:
P3:
```suggestion
                case .updateViewController(let title):
```
이게 자연스러워 보입니당 !

Reply:
요상한 자동완성이.... 반영하겠읍니다!